manifests/p/PHP/PHP/8/2/8.2.26/PHP.PHP.8.2.installer.yaml

```diff
@@ -0,0 +1,32 @@
+# Created with PHPWatch/winget-pkgs - https://github.com/PHPWatch/php-winget-manifest
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: PHP.PHP.8.2
+PackageVersion: 8.2.26
+InstallerLocale: en-US
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+  - RelativeFilePath: php.exe
+    PortableCommandAlias: php
+Commands:
+  - php
+  - php82
```

Comment:
Just leaving a note - reading this feels like that you may want WinGet to create multiple aliases (`php` & `php82`) for the same `php.exe` executable added to PATH. That functionality isn't in the CLI implementation yet. You may want to upvote

- https://github.com/microsoft/winget-cli/issues/2884

if you feel the PHP package can make use of multiple command aliases